WordPress/WordPressTest/ItemProviderMediaExporterTests.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import OHHTTPStubs
+import XCTest
+@testable import WordPress
+
+final class ItemProviderMediaExporterTests: XCTestCase {
```

Comment:
`MediaImageExporterTests` that’s used internally already has a bunch of tests, so we mostly rely on them and just test the integration.